contracts/contracts/Job.sol

```diff
@@ -398,10 +410,6 @@ contract Job is IJob, Ownable {
         MINIMUM_BOUNTY = newValue;
     }
 
-    function setDefaultDeposit(uint256 newValue) external onlyOwner {
```

Comment:
I removed this since it was no longer relevant and was causing a compilation error.